.github/workflows/build_and_test.yml

```
@@ -9,7 +9,10 @@ on:
 jobs:
   build_and_test_with_code_coverage:
     name: Build, test and upload code coverage
```

```suggestion
    name: Build and test
```
Let's call it simply `Build and test`, also I'll update required checks (they are set by name) so that this PR can be merged